docs/modules/ROOT/pages/index.adoc

```diff
@@ -632,6 +639,19 @@ If the *<<_identity_assertion,identity assertion>>* contains an `expected_claim_
 
 The resulting hash MUST exactly match the value present in `expected_claim_generator.hash`. The `cawg.identity.expected_claim_generator.mismatch` error code SHALL be used to report assertions that contain other values.
 
+==== Validating the `expected_countersigners` value
+
+If the *<<_identity_assertion,identity assertion>>* contains an `expected_countersigners` value in `signer_payload`, it MUST be validated as follows:
```

Comment:
contains an `expected_countersigners` *field* in `signer_payload`, it MUST be validated as follows: